analyzer/src/main/kotlin/managers/Yarn2.kt

```diff
@@ -338,55 +338,55 @@ class Yarn2(
                 mapping += it.value
             }
         }
-        graphBuilder.addPackages(allPackages.values)
```

Comment:
Good catch. This was never required, since the calls to `addDependency()` automatically add the packages to the graph builder. This would only be needed if packages were constructed in an alternative way, which is not the case here.